src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -393,6 +398,7 @@
   (let [handler (RequestLogHandler.)
         logger (RequestLogImpl.)]
     (.setFileName logger (:access-log-config config))
+    (.setQuiet logger true)
```

Comment:
This gets us to 100% compliance for Wolfe's Principle 1 on keeping logging silent for test runs - for now.  Without this change, a few verbose logback "-INFO" messages are written out when the logger is initialized, like these:

```
11:55:59,250 |-INFO in LogbackRequestLog - Will use configuration file [./dev-resources/puppetlabs/trapperkeeper/services/webserver/request-logging.xml]
11:55:59,254 |-INFO in ch.qos.logback.access.joran.action.ConfigurationAction - debug attribute not set
11:55:59,254 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [appender.TestListAppender]
11:55:59,254 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [LIST]
```

These don't seem particularly useful and probably not worth making configurable for now.